lib/chef/provider/package/dpkg.rb

```diff
@@ -66,6 +66,7 @@ def load_current_resource
                   if pkginfo = DPKG_INFO.match(line)
```

Comment:
This is using popen4, if there's not already a bug open on fixing this, we should file one.